dev/releases/HelpLinks-to-JSON.g

```diff
@@ -0,0 +1,84 @@
+##  A utility for dumping links to all available manual sections
+##  (for use with named manual references on web pages or in search utilities).
+##
+##  The main purpose for this script is to create the help-links.json file that
+##  is used by the GAP website. This happens as part of the release process.
+##
+##  In order to be used on the GAP website, relative to a particular GAP
+##  version, this script should be run from within the corresponding released
+##  version of GAP, with exactly its bundled packages (installed in the pkg/
+##  subdirectory), and with no special user config. The GAP and package manuals
+##  are precompiled in the released version.
+
+# Temporary hack to make sure that the record components in the JSON file are
+# sorted, to increase stability in the output between GAP versions.
+InstallMethod(RecNames, [IsRecord and IsInternalRep], x -> AsSSortedList(REC_NAMES(x)));
```

Comment:
See https://github.com/gap-packages/json/pull/24 - I hope @ChrisJefferson will merge that and make a new release soon

Reply:
I added a comment to the script explicitly pointing to this PR. I also added the same hack and comment to the package-infos script as well.